operator/text/v0/convert.go

```diff
@@ -71,3 +69,57 @@ func convertToText(input ConvertToTextInput) (ConvertToTextOutput, error) {
 		Error: res.Error,
 	}, nil
 }
+
+type uft8EncodedFileConverter struct{}
+
+func (m uft8EncodedFileConverter) convert(contentType string, b []byte) (ConvertToTextOutput, error) {
+
+	before := time.Now()
+	content := string(b)
+
+	duration := time.Since(before)
+	millis := duration.Milliseconds()
+
+	metadata := map[string]string{}
+
+	return ConvertToTextOutput{
+		Body:  content,
+		Meta:  metadata,
+		MSecs: uint32(millis),
+		Error: "",
+	}, nil
+}
+
+func isSupportedByDocconv(contentType string) bool {
+	return contentType != "application/octet-stream"
+}
+
+func convertToText(input ConvertToTextInput) (ConvertToTextOutput, error) {
+
+	contentType, err := getContentTypeFromBase64(input.Doc)
+	if err != nil {
+		return ConvertToTextOutput{}, err
+	}
+
+	b, err := base64.StdEncoding.DecodeString(base.TrimBase64Mime(input.Doc))
+	if err != nil {
+		return ConvertToTextOutput{}, err
+	}
+
+	// TODO: support xlsx file type with https://github.com/qax-os/excelize
+	var converter converter
+	if isSupportedByDocconv(contentType) {
```

Comment:
We need to explicitly compare if the `contentType` is included in the list of supported types by `docconv` [here](https://github.com/sajari/docconv/blob/785a29a00de4b976c379fd38299c220307220684/docconv.go#L58). 
`contentType != "application/octet-stream"` doesn't mean it can be supported by `docconv`

Reply:
@donch1989 
Do you mean like [this](https://github.com/instill-ai/component/pull/149/commits/13a32fb4910af033495c393c7ceb43ddde0f80fd)?

Reply:
Yes, and you can use a map for better performance.